Dockerfile

```diff
@@ -0,0 +1,62 @@
+# ./Dockerfile
+
+FROM rockylinux:8.5
+
+# Host dependencies 
+RUN yum -y upgrade && yum -y install \
+    bc \
+    bzip2 \
+    cpio \
+    dbus-x11 \
+    diffutils \
+    epel-release \
+    expat-devel \
+    git \
+    glibc-devel \
+    libffi-devel \
+    libjpeg-turbo-devel \
+    ncurses-compat-libs \
+    openssl-devel \
+    patch \
+    python3-devel \
+    python3-setuptools \ 
+    readline-devel \
+    unzip \ 
+    xorg-x11-server-Xvfb \
+    xorg-x11-utils \
+    xz \
+    zlib-devel
+
+
+RUN yum -y group install "Development Tools"
+
+# Get fakeroot which needs epel-release 
+RUN yum -y install fakeroot
+
+# Copy in scripts and dls rootfs, annotypes, pymalcolm, and malcolmjs
+COPY PandABlocks-rootfs/.github/scripts /scripts
+COPY rootfs /rootfs
+COPY annotypes /annotypes
+COPY pymalcolm /pymalcolm
+COPY malcolmjs /malcolmjs
+
+# Toolchains and tar files
+RUN bash scripts/GNU-toolchain.sh
+RUN bash scripts/tar-files.sh
+
+# For the documentation
+RUN pip3 install matplotlib \ 
+    rst2pdf \
+    sphinx \
+    sphinx-rtd-theme \
+    --upgrade docutils==0.16
+
+# Create config file for dls-rootfs
+RUN bash scripts/config-file-rootfs.sh
```

Comment:
Isn't this `ln -s CONFIG.example CONFIG`?

Reply:
This is creating the CONFIG.local file for the checked-out [rootfs ](https://github.com/dls-controls/rootfs) repo which it builds into the container. As that repo is not just used for PandA I thought we may not want to update its CONFIG.example with the PandA Container paths so I kept its creation from this script. If it's a better solution I can use its CONFIG.example and then update the paths?

Reply:
Ahh, I forgot we needed a CONFIG for rootfs too. This is fine then